portfolio/src/main/webapp/script.js

```diff
@@ -60,4 +60,10 @@ function four() {
     elements[i].style.msFlex = "25%";  // IE10
     elements[i].style.flex = "25%";
   }
+}
+
+async function hiLeah() {
+  const response = await fetch('/data');
```

Comment:
You might consider using promises, so you're not blocked waiting for the network request to complete. This looks like a pretty good article covering this: https://github.com/gwynethsc/step/blob/master/portfolio/src/main/webapp/script.js#L56